examples/aws-cloudwatch/capacity_planning/ec2.juttle

```diff
@@ -22,7 +22,7 @@ const instance_prices = {
     't2.medium': 0.05,
 };
 
-read mysql -table 'aws_aggregation' -from :0: -to :end:
+read mysql -table 'aws_aggregation' -last :1h:
```

Comment:
is this an intentional change? doesn't seem to have anything to do with filtering.